wremnants/datasets/datasetDict_gen.py

```diff
@@ -195,55 +195,55 @@
     genDataDict.update({
         'ZToMuMu_horace-v3-lo-photos' : { 
                 'filepaths' :
-                ["/eos/cms/store/cmst3/group/wmass/w-mass-13TeV/NanoGen/Horace_v3/ZToMuMu_TuneCP5_13TeV-horace-born-fsr-photos_nopair-isr-pythia/job*.root"],
+                ["root://eoscms.cern.ch//store/cmst3/group/wmass/w-mass-13TeV/NanoGen/Horace_v3/ZToMuMu_TuneCP5_13TeV-horace-born-fsr-photos_nopair-isr-pythia"],
                 'xsec' : xsec_powheg_ZToMuMu_LO,
                 'group': "Zmumu",
         },
         'ZToMuMu_horace-v3-qed' : { 
                 'filepaths' :
-                ["/eos/cms/store/cmst3/group/wmass/w-mass-13TeV/NanoGen/Horace_v3/ZToMuMu_TuneCP5_13TeV-horace-exp-old-fsr-off-isr-pythia/job*.root"],
+                ["root://eoscms.cern.ch//store/cmst3/group/wmass/w-mass-13TeV/NanoGen/Horace_v3/ZToMuMu_TuneCP5_13TeV-horace-exp-old-fsr-off-isr-pythia"],
                 'xsec' : xsec_ZmmPostVFP,
                 'group': "Zmumu",
         },
         'ZToMuMu_horace-v3-nlo' : { 
                 'filepaths' :
-                ["/eos/cms/store/cmst3/group/wmass/w-mass-13TeV/NanoGen/Horace_v3/ZToMuMu_TuneCP5_13TeV-horace-exp-new-fsr-off-isr-off/job*.root"],
+                ["root://eoscms.cern.ch//store/cmst3/group/wmass/w-mass-13TeV/NanoGen/Horace_v3/ZToMuMu_TuneCP5_13TeV-horace-exp-new-fsr-off-isr-off"],
                 'xsec' : xsec_powheg_ZToMuMu_NLOEW,
                 'group': "Zmumu",
         },
         'WplusToMuNu_horace-v3-lo-photos' : { 
                 'filepaths' :
-                ["/eos/cms/store/cmst3/group/wmass/w-mass-13TeV/NanoGen/Horace_v3/WplusToMuNu_TuneCP5_13TeV-horace-born-fsr-photos_nopair-isr-pythia/job*.root"],
+                ["root://eoscms.cern.ch//store/cmst3/group/wmass/w-mass-13TeV/NanoGen/Horace_v3/WplusToMuNu_TuneCP5_13TeV-horace-born-fsr-photos_nopair-isr-pythia"],
```

Comment:
I think they only exist on eos, but I don't see a good reason for these to have the full eospath rather than {BASE_PATH}. I know you didn't do this in this PR, but just thinking

Reply:
These samples don't actually live under the common path with the rest, so a bit more reorganization would be needed to deal with this

ie
the usual base_path would be ```root://eoscms.cern.ch//store/cmst3/group/wmass/w-mass-13TeV/NanoAOD``` but these live under ```root://eoscms.cern.ch//store/cmst3/group/wmass/w-mass-13TeV/NanoGen```